### PR TITLE
test(upgrade): fix up tests/code to work with ivy

### PR DESCRIPTION
**Pull request by petebacondarwin**
All failing tests are now disabled with a reasonable cause description.

### CODE REVIEW

File: packages/upgrade/test/common/downgrade_component_adapter_spec.ts [hunk 1]
```diff
@@ -177,26 +178,28 @@ withEachNg1Version(() => {
...
-      it('should remove the testability hook when destroy a component', () => {
-        const registry = TestBed.get(TestabilityRegistry);
-        expect(registry.getAllTestabilities().length).toEqual(0);
-        adapter.createComponent([]);
-        expect(registry.getAllTestabilities().length).toEqual(1);
-        adapter.registerCleanup();
-        element.remove !();
-        expect(registry.getAllTestabilities().length).toEqual(0);
-      });
+      fixmeIvy('Runtime compiler is not loaded') &&
```
Comment (kara) [inline packages/upgrade/test/common/downgrade_component_adapter_spec.ts]: Would you mind prefixing these root causes with the Jira ticket number? e.g. 
```suggestion
      fixmeIvy('FW-561: Runtime compiler is not loaded') &&
```

File: packages/upgrade/test/static/integration/change_detection_spec.ts [hunk 2]
```diff
@@ -75,57 +76,60 @@ withEachNg1Version(() => {
...
-
-
-         const element = html('<my-app></my-app>');
-
-         bootstrap(platformBrowserDynamic(), Ng2Module, element, ng1Module).then((upgrade) => {
-           appComponent.value = 5;
-         });
-       }));
+    fixmeIvy(
+        'Rendering is being run on next "animation frame" rather than "Zone.microTaskEmpty" trigger') &&
```
Comment (kara) [inline packages/upgrade/test/static/integration/change_detection_spec.ts]: Same here - can you add the Jira issue number?

**Review by kara — CHANGES_REQUESTED** [review]
Looks great, just needs Jira issue numbers on root causes

File: packages/upgrade/test/common/downgrade_component_adapter_spec.ts (hunk 1 above)
Comment (petebacondarwin) [reply packages/upgrade/test/common/downgrade_component_adapter_spec.ts]: Done

**Review by kara — APPROVED** [review]
LGTM

**Pull request closed without merging**